### PR TITLE
Create product-lead-responsibilities.md

### PR DESCRIPTION
**Pull request by laurjpeterson**
Writing up product lead responsibilities for the team to see.

### CODE REVIEW

File: docs/product-lead-responsibilities.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Overall inspiration & focus
+* Keep the Veteran at the center of team and stakeholder decisions
+* Deeply understand problems to solve, so Caseflow builds the right product or feature at the right time for our users (Veterans) and stakeholders
+* Foster communication and collaboration among all disciplines
+* Organize team members and stakeholders for collaboration and coordination across teams, disciplines, and products. Break down silos. 
+* Promote design research with users and the business to uncover needs and problems to solve (note: can conduct research independently)
+* Execute Caseflow’s product strategy; ship products and features into production
+* Measure success - defining Caseflow-wide success metrics as well as feature-level measures
+
+## Team leadership & culture*
```
Comment (pkarman) [inline docs/product-lead-responsibilities.md]: trailing `*` in the head?

File: docs/product-lead-responsibilities.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Co-facilitate conversations around team structure, norms, standards, and processes with other discipline leads
+* Foster a culture of continuous improvement, trust, and feedback
+* Manage/mentor/coach/hire Caseflow product managers
+	* Note: hiring for contractors only, not VA employees
+* Initiate and facilitate Caseflow Feedback cycles
+* Answers questions about team structure and role definition
+* Collaboration with other disciplines, including design, engineering, project management, support, training, etc.
+
+## Vision and strategy*
```
Comment (pkarman) [inline docs/product-lead-responsibilities.md]: trailing `*` in the head?

File: docs/product-lead-responsibilities.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* Execute Caseflow’s product strategy; ship products and features into production
+* Measure success - defining Caseflow-wide success metrics as well as feature-level measures
+
+## Team leadership & culture*
+* Co-facilitate conversations around team structure, norms, standards, and processes with other discipline leads
+* Foster a culture of continuous improvement, trust, and feedback
+* Manage/mentor/coach/hire Caseflow product managers
+	* Note: hiring for contractors only, not VA employees
+* Initiate and facilitate Caseflow Feedback cycles
+* Answers questions about team structure and role definition
```
Comment (pkarman) [inline docs/product-lead-responsibilities.md]: is answering those questions really a job responsibility? or is facilitating the conversation and documenting the results? If you really spend that much time answering questions, we should be thinking about how to make it more self-serve.

File: docs/product-lead-responsibilities.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+* Prioritize projects and initiatives across Caseflow. Conduct prioritization discussions initially with Caseflow engineering and design leads.
+* Facilitate prioritization discussions with stakeholders using the prioritization framework. Propose and articulate product priorities to stakeholders.
+	* E.g. stakeholders request a new project. Ensure the prioritization framework has been filled out. Explain what initiatives need to be deprioritized if the new work is prioritized. 
+* Oversee roadmapping activities among product managers, and ensure they are focusing on the highest leverage initiatives to achieve Caseflow-wide product vision
+* From user and business research, define the projects (epics) and user stories that are ultimately designed and implemented. Initiate then delegate and oversee the writing of product briefs/project outlines in order to align stakeholders, designers, engineers, project managers, support, training, and any other team members not mentions.
```
Comment (pkarman) [inline docs/product-lead-responsibilities.md]: last word should be "mentioned"

File: docs/product-lead-responsibilities.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	* Note: engineers and designers also define projects and user stories
+	* Note: this delegation depends on team staffing and bandwidth
+* In collaboration with other product managers, follow this general process for each project/epic:
```
Comment (pkarman) [inline docs/product-lead-responsibilities.md]: this section doesn't belong in the job duties doc, IMO. Too specific. Maybe instead: "Define and document process each project/epic will follow."

File: docs/product-lead-responsibilities.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+* Sprint-by-sprint activities:
+	* Define sprint goals
+	* Decide sprint initiatives and tasks with design and engineering
+  
+## Product support
+* Problem solve support tickets with tech leads and others 
+* Regular 1:1 with product support team lead to ensure they are supported
+
+## Stakeholder management and communication
+* Owns relationship, regular updates, and expectation setting with Office of Information & Technology	* Maintain a regular cadence of meetings
```
Comment (pkarman) [inline docs/product-lead-responsibilities.md]: is this 2 bullet points?

File: docs/product-lead-responsibilities.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
+# Caseflow product lead responsibilities
+
+## Overall inspiration & focus
+* Keep the Veteran at the center of team and stakeholder decisions
+* Deeply understand problems to solve, so Caseflow builds the right product or feature at the right time for our users (Veterans) and stakeholders
```
[the remark this answers is not in the document]
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: indirectly, yes! I think we're serving them at the end of the day. Serving our users (VA employees) helps us serve Veterans

File: docs/product-lead-responsibilities.md (hunk 6 above)
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: fixed!

File: docs/product-lead-responsibilities.md (hunk 1 above)
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: fixed

File: docs/product-lead-responsibilities.md (hunk 3 above)
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: Makes sense - changed the language to reflect facilitating the conversations about this

File: docs/product-lead-responsibilities.md (hunk 2 above)
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: fixed

File: docs/product-lead-responsibilities.md (hunk 4 above)
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: fixed

File: docs/product-lead-responsibilities.md (hunk 5 above)
Comment (laurjpeterson) [reply docs/product-lead-responsibilities.md]: good idea - pulled it out

**Pull request merged**